### PR TITLE
Refactor you sure component

### PR DESCRIPTION
**Pull request by WarrenK-design**
Refactored the are you sure component on the trade page based on feedback. 

Simpler design doesn't show as much information by default, but the information is still there if they want it in the dropdown. 

**NOTE** Need to so an NPM install, new package installed thats why so many lines changed

# Initial Page 
## New Design 
@beckskelly stole bitta your code for images! 
image
image

## Old Design 
image

# Success Page
## New Design
Tried to make bit more fun added icons, confetti falls on successful order. Flow a bit better too you can go to the leader board or the stocks page. Note when you are resizing the confetti defaults to your original size so if messing between mobile and desktop may notice size difference.  Have option then to either go and browse more stocks or see the leader board. 
image
## Old Design
image

### CODE REVIEW

**Review by joeycor92 — APPROVED** [review]
As discussed and from a few different iterations between us, it doesn't really seem like it can be improved too much. I'm happy to approve.

**Comment by joeycor92** [comment]
Happy with the new change that we discussed in the chat. Agree that the new spending power stands out a lot more!

File: src/components/confirmOrderComponents/AreYouSure.js [hunk 1]
```diff
@@ -73,31 +74,54 @@ function AreYouSure({showState,setShowState,portfolioId,stockId,buyOrSell,orderT
...
-        </Modal.Header>
+        <Modal centered show={showState} onHide={handleClose} backdrop="static">
+            <Row md={1} sm={1} xs={1} className="textCenter pt-1">
+                <Modal.Title className="bolded">{stockName}</Modal.Title>
+                <h5 className="newSpendingPowerSubtitle">New Spending Power {parseFloat(newPortfolioBalance).toLocaleString('en-US', {style: 'currency', currency: 'USD'})}</h5>
+            </Row>
         <Modal.Body>
             {error && <MessageAlert variant="danger">{error}</MessageAlert>}
-            {success && <MessageAlert variant="success">{success}</MessageAlert>}
+            {/* {success && <MessageAlert variant="success">{success}</MessageAlert>} */}
```
Comment (joeycor92) [inline src/components/confirmOrderComponents/AreYouSure.js:84]: Actually meant to say to take out this commented out code here.

**Review by joeycor92 — CHANGES_REQUESTED** [review]

**Review by beckskelly — APPROVED** [review]
Sorry bit late to the party here, looks great @WarrenK-design and works really well. Love the confetti

File: src/components/confirmOrderComponents/AreYouSure.js (hunk 1 above)
Comment (WarrenK-design) [reply src/components/confirmOrderComponents/AreYouSure.js]: Fixed that there cheers!

**Review by joeycor92 — APPROVED** [review]
Thanks, approved.

**Pull request merged**